packages/Textarea/src/textarea.scss

```diff
@@ -2,94 +2,145 @@
-  font-size: var(--ids-textarea-font-size);
-  position: relative;
-
-  &,
-  & * {
-    box-sizing: border-box;
-  }
-
-  &__field {
-    border: .1rem solid var(--ids-textarea-border-color);
-    border-radius: var(--ids-textarea-border-radius);
-    color: var(--ids-textarea-color);
-    display: block;
-    font-family: inherit;
-    font-size: inherit;
-    line-height: var(--ids-textarea-line-height);
-    outline: none;
-    padding: var(--ids-textarea-padding);
-    resize: none;
-    width: 100%;
-
-    @include mixins.fancy-webkit-scrollbar;
-
-    &::placeholder {
-      color: var(--ids-textarea-color-placeholder);
-    }
+    font-family: var(--ids-textarea-font-family);
+    font-size: var(--ids-textarea-font-size);
+    position: relative;
 
-    &:focus,
-    .ids-textarea--focus & {
-      border-color: var(--ids-textarea-focus-border-color);
+    &,
+    & * {
+        box-sizing: border-box;
     }
 
-    &--error {
-      border-color: var(--ids-textarea-error-border-color);
+    &__field {
+        border: .1rem solid var(--ids-textarea-border-color);
+        border-radius: var(--ids-textarea-border-radius);
+        color: var(--ids-textarea-color);
+        display: block;
+        font-family: inherit;
+        font-size: inherit;
+        line-height: var(--ids-textarea-line-height);
+        outline: none;
+        padding: var(--ids-textarea-padding);
+        resize: none;
+        width: 100%;
+
+        @include mixins.fancy-webkit-scrollbar;
+
+        &::placeholder {
+            color: var(--ids-textarea-color-placeholder);
+        }
+
+        &:focus:not(&--error),
+        .ids-textarea--focus:not(&--error) & {
+            border-color: var(--ids-textarea-focus-border-color);
+        }
+
+        &:hover:not(:disabled) {
+            background-color: var(--ids-textarea-hover-background-color);
+        }
+
+        &:hover:not(:disabled, &--error) {
+            border-color: var(--ids-textarea-hover-border-color);
+        }
+
+        &--error {
+            border-color: var(--ids-textarea-error-border-color);
+        }
+
+        &:disabled {
+            background: var(--ids-textarea-disabled-background);
+            border-color: var(--ids-textarea-disabled-background);
+            color: var(--ids-textarea-disabled-color);
+            cursor: not-allowed;
+        }
     }
 
-    &:disabled {
-      background: var(--ids-textarea-disabled-background);
-      border-color: var(--ids-textarea-disabled-background);
-      color: var(--ids-textarea-disabled-color);
-      cursor: not-allowed;
+    &__char-indicator {
+        bottom: var(--ids-textarea-char-indicator-bottom);
+        color: var(--ids-textarea-char-indicator-color);
+        font-size: var(--ids-textarea-char-indicator-size);
+        line-height: var(--ids-textarea-char-indicator-line-height);
+        position: absolute;
+        right: var(--ids-textarea-char-indicator-right);
     }
+}
 
-    &--has-char-count {
```

Comment:
why was this removed?

Reply:
It was there to prevent this overlap:
![image](https://github.com/gsoft-inc/ov-igloo-ui/assets/110498164/12874c69-3920-45cc-b283-4b011624c089)